src/bindgen/language_backend/clike.rs

```diff
@@ -911,30 +911,42 @@ impl LanguageBackend for CLikeLanguageBackend<'_> {
                 fields,
                 path,
             } => {
+                let is_constexpr = self.config.language == Language::Cxx
+                    && (self.config.constant.allow_constexpr || l.can_be_constexpr());
```

Comment:
This seems wrong? Shouldn't it be `allow_constexpr && can_be_constexpr`?

Reply:
Oh.. I was confused by allow_static_const. Thank you